charter.md

```diff
@@ -1,134 +1,123 @@
-Member | Member | Tomoaki Hori | STELAB
-Member | Member | Nozomu Nishitani | STELAB
-Member | Aurélie Marchaudon | IRAP/CNRS
-Member | Member | Nigel Wade | Leicester
-Member | Member | Tsutomu Nagatsuma | NICT
-Member | Keisuke Hosokawa | DICE
-Member | Ethan Miller | JHU/APL
-Member | Member | Ermanno Amata | IAPS-INAF
-Member | Tom Kane | LaTrobe
-Member | Hong-Qiao Hu | PRIC
-Member | Xavier Vallieres | CNRS
-Member | Chris Thomas | Leicester
-Member | Manabu Kunitake | NICT
-Observer | Mike Ruohoniemi | Virginia Tech
-Observer | Kathryn McWilliams | University of Saskatchewan
-Observer | Dieter Andre | University of Saskatchewan
-Observer | Jean-Pierre St.Maurice | University of Saskatchewan
+Responsibilty | Name | Institution | Radars
+--- | --- | --- | ---
+Chair | Kevin Krieger | USASK | sas, pgr, rkn, inv, cly
+Member | Marci Detwiller | USASK | sas, pgr, rkn, inv, cly
+Member | Marina Schmidt | USASK | sas, pgr, rkn, inv, cly
+Member | Keith Kotyk | USASK | sas, pgr, rkn, inv, cly
+Member | Kevin Sterne | VT | bks, kap, gbr, fhe, fhw
+Member | Simon Shepherd | Dartmouth | cve, cvw
+Member | Evan Thomas | Dartmouth | cve, cvw
+Member | Akira Sessai Yukimatu | NIPR | sye, sys
+Member | Jordan Wiker | JHU/APL | wal
+Member | Enrico Simeoli | IAPS/INAF | dce, dcn
+Member | Kehe Wang | BOM
+Member | John Devlin | LaTrobe | tig, unw, bpk
+Member | Jim Whittington | LaTrobe | tig, unw, bpk
+Member | Mark Gentile | LaTrobe | tig, unw, bpk
+Member | Andrew McDonald | LaTrobe | tig, unw, bpk
+Member | Paul Breen | BAS | fir
+Member | Tim Barnes | BAS | fir
+Member | Bill Bristow | UAF | sps, mcm, ade, adw, kod
+Member | Jianjun Liu | PRIC | zho
+Member | Hong-Qiao Hu | PRIC | zho
+Member | Judy Stephenson | Durban | san
+Member | Tomoaki Hori | STELAB | hok, hkw
+Member | Nozomu Nishitani | STELAB | hok, hkw
+Member | Aurélie Marchaudon | IRAP | ker
+Member | Tsutomu Nagatsuma | NICT | ksr
+Member | Keisuke Hosokawa | DICE/UEC |
+Member | Chris Thomas | Leicester | han, pyk, sto
+Member | Tim Yeoman | Leicester | han, pyk, sto
+Member | Qing-He Zhang | SDU
+Member | Emma Bland | UNIS | lyr 
+Member | Mikko Syrjäsuo| UNIS | lyr
+Observer | Mike Ruohoniemi | VT
+Observer | Kathryn McWilliams | USASK
+Observer | Dieter Andre | USASK
+Observer | Jean-Pierre St.Maurice | USASK
+Observer | Federica Marcucci | IAPS/INAF 
 
 ## Communications
 
-Communication shall primarily be done over email via the Data-Distribution Working-Group (Darn-ddwg)
-mailing list set up by Julian Thornhill, maintained by Leicester University. To send email to the 
-list, send mail to <darn-ddwg@ion.le.ac.uk>.
+Communication shall primarily be done over email via the DDWG mailing list set up by Nozomu Nishitani,
+maintained at Nagoya University. To send email to the list, send mail to <darn-ddwg@isee.nagoya-u.ac.jp>.
+Previously, a mailing list set up by Julian Thornhill, maintained by Leicester University, was used. 
+That list address was <darn-ddwg@ion.le.ac.uk>.
```

Comment:
Should you also include GitHub as a way to bring up issues as a form of communication?

Reply:
Yes, I will add in some sections for:
Raising issues
Meetings
other policies